src/test/java/org/codehaus/plexus/util/xml/pull/MXSerializerTest.java

```diff
@@ -55,4 +56,17 @@ private String expectedOutput() {
         out.append("</root>");
         return out.toString();
     }
+
+    /**
+     * Tests MJAVADOC-793.
+     */
+    @Test
+    public void testMJAVADOC793() throws IOException {
+        // should be no-ops
+        new MXSerializer().writeElementContent(null, null);
+        new MXSerializer().writeAttributeValue(null, null);
+        final StringWriter stringWriter = new StringWriter();
+        new MXSerializer().writeElementContent(null, stringWriter);
+        new MXSerializer().writeAttributeValue(null, stringWriter);
```

Comment:
should support ?

```java
new MXSerializer().writeElementContent("value", null);
new MXSerializer().writeAttributeValue("value", null);
```

Reply:
Also add some of assertions for what content of `stringWriter` should be.

Reply:
Done.